crates/sui-types/src/messages.rs

```diff
@@ -2481,6 +2483,20 @@ pub enum CommandArgumentError {
     InvalidObjectByMutRef,
 }
 
+#[derive(Eq, PartialEq, Clone, Debug, Serialize, Deserialize, Hash, Error)]
+pub enum PackageUpgradeError {
```

Comment:
Sorry... I might have blown this enum away

Reply:
No worries :) I managed to resurrect it with no issues :)